### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by blankey1337**

### CODE REVIEW

**Comment by harry1453** [comment]
@ohager Sorry, it dismissed your review because it's set to do that against `master` (but not `develop`)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,27 +1,10 @@
-## For outside contributors
+## Contributing
 
-Please use the following workflow:
+Please use the following workflow to contribute:
 
 * Fork the repository
+* Create a branch which accurately describes your fix or feature (prefix branches with `fix/`, `feat/`, `docs/`, etc). 
+** Please use signed commits if you can - you are contributing to the backbone of a cryptocurrency.
```
[the remark this answers is not in the document]
Comment (blankey1337) [reply CONTRIBUTING.md]: That is a sub-bullet. Is that not valid markdown?

[the remark this answers is not in the document]
Comment (harry1453) [reply CONTRIBUTING.md]: Nope, if you checked the page at the time it looked like:

* Create a branch which accurately describes your fix or feature (prefix branches with `fix/`, `feat/`, `docs/`, etc). 
** Please use signed commits if you can - you are contributing to the backbone of a cryptocurrency.

[the remark this answers is not in the document]
Comment (harry1453) [reply CONTRIBUTING.md]: I fixed the sub-bullet :)

**Pull request merged**